string start_floor
string destination_floor
string start_area
string destination_area
string start_local_area
string destination_local_area
geometry_msgs/Point start_position
string destination_task
---
---
string current_navigation_skill
string current_goal
int32 skill_state